src/style.css

```diff
@@ -0,0 +1,63 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", serif;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
```

Comment:
what if you have another `<ul>` and `<a>` on the page and you don't want to reset this style for them?...

Reply:
done